src/components/__tests__/AggregateList-test.tsx

```diff
@@ -12,6 +12,8 @@ describe("AggregateList", () => {
   let data;
   let productionLibrary1 = modifyLibrary(testLibrary1, { "name": "Production Library 1", "registry_stage": "production" });
   let productionLibrary2 = modifyLibrary(productionLibrary1, { "name": "Production Library 2" });
+  productionLibrary2.uuid = "UUID2";
+
```

Comment:
You can change the UUID in the `modifyLibrary` call; just pass it in as another key: value pair.

Reply:
`let productionLibrary2 = modifyLibrary(productionLibrary1, { "uuid": "uuid2", "name": "Production Library 2" });` ? I tried that. I mentioned that the root properties don't get updated in the object but the children objects do.

Reply:
Eeeeeeek.  I will fix that.  That is not the intended behavior.

Reply:
Here you go! https://github.com/NYPL-Simplified/registry_admin/pull/44

Reply:
Thanks.